src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -122,6 +122,23 @@ public FeedbackSessionResultsBundle(FeedbackSessionAttributes feedbackSession,
         // roster.*Table is populated using the CourseRoster data directly
         this.rosterTeamNameMembersTable = getTeamNameToEmailsTableFromRoster(roster);
         this.rosterSectionTeamNameTable = getSectionToTeamNamesFromRoster(roster);
+        if (!isAllQuestionNumbersConsistent()) {
+            log.severe(feedbackSession.getIdentificationString() + " has invalid question numbers");
+        }
+    }
+    
+
+    //TODO can be removed once we are sure that question numbers will be consistent
+    private boolean isAllQuestionNumbersConsistent() {
+        Set<Integer> questionNumbersInSession = new HashSet<>();
+        for (FeedbackQuestionAttributes question : this.questions.values()) {
```

Comment:
Oh, I remembered why I initially just did the check for duplicate question numbers now. 

At this point in the code, it's no longer possible to distinguish between loading all questions, and loading a single question by ajax. Now the ajax loading will always trigger the log.severe because it only has a single question.